integerDetection.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TEXT=$(cat $1)
+
+for WORD in $TEXT
+do
+    if [[ $WORD =~ ^[-]?[0-9]+[.]?$ ]]; then
```

Comment:
It doesnt consider numbers before commas. i.e. "I was born in **2000,** I'm 22 years old"